### PR TITLE
feat(linux): Don't create Sentry events for errors

### PR DESCRIPTION
**Pull request by ermshiperete**
We don't want to send events to Sentry for errors that we handled.

@keymanapp-test-bot skip

### CODE REVIEW

**Comment by ermshiperete** [comment]
Another option to get rid of the handled errors on Sentry would be to use `logging.error()` instead of `logging.critical()`.

File: linux/keyman-config/keyman_config/__init__.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def _(txt):
 
         sentry_logging = LoggingIntegration(
             level=logging.INFO,           # Capture info and above as breadcrumbs
-            event_level=logging.CRITICAL  # Send critical errors as events
+            event_level=None              # Don't create events for errors
```
Comment (rc-swag) [inline linux/keyman-config/keyman_config/__init__.py:58]: Won't this mean all no events are sent to sentry even unhandled events?

Comment (ermshiperete) [reply linux/keyman-config/keyman_config/__init__.py]: What was I thinking??? Of course you're right.

Comment (ermshiperete) [reply linux/keyman-config/keyman_config/__init__.py]: done.

**Review by mcdurdin — APPROVED** [review]

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.175-alpha](https://keyman.com/downloads/releases/alpha/15.0.175)

**Pull request merged**